Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift

```diff
@@ -163,12 +164,12 @@ extension DaysUntilBirthdayUITests_iOS {
       .staticTexts[passwordManagerPrompt]
       .waitForExistence(timeout: timeout) {
       guard sampleApp
-        .buttons["Not Now"]
+        .buttons[nowNowText]
         .waitForExistence(timeout: timeout) else {
-        XCTFail("Failed to find 'Not Now' button")
+        XCTFail(String(format: "Failed to find '%@' button", nowNowText))
```

Comment:
I think string interpolation is more succinct: `XCTFail("Failed to find \(notNowText) button.")`.